core/src/main/scala/scalaz/Void.scala

```diff
@@ -1,29 +1,15 @@
 package scalaz
 
-import Liskov.<~<
+import scalaz.zio.{Void => ZVoid}
```

Comment:
`ZioVoid` maybe? we are in scalaz, just the Z ain't enough :slightly_smiling_face: